shop-data/src/main/java/com/maxshop/response/CategoryResponse.kt

```diff
@@ -0,0 +1,3 @@
+package com.maxshop.response
+
+internal data class CategoryResponse(val categoryName: String)
```

Comment:
Please remove `CategoryReposnse` and `CategoryResponseMapper`, because they are not used

Reply:
fixed